workflows/prognostic_scream_run/setup.py

```diff
@@ -23,6 +23,8 @@
     entry_points={
         "console_scripts": [
             "write_scream_run_directory=scream_run.cli:write_scream_run_directory",
+            "scream_run=scream_run.cli:scream_run",
```

Comment:
Having several commands that do related things in this package (prepare config, write run directory, run) might be better organized under a single CLI command with subcommands, similar to how `runfv3` has `create`, `append` etc.

Reply:
Consolidated into `scream_run` : `prepare`, `execute`.